packages/ui/src/globals.css

```
@@ -21,31 +45,291 @@
+  :where(.dark, .dark *) {
+    --tw-shadow-color: rgb(255 255 255 / 0.06);
+  }
+
+  html,
+  body,
+  #root {
+    height: 100%;
+  }
+
+  body {
+    font-family: 'Inter', ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)